apps/typegpu-docs/src/content/docs/fundamentals/bind-groups.mdx

```diff
@@ -133,11 +133,12 @@ Runtime-sized arrays are only usable with **storage** buffers.
 const Filter = (n: number) =>
   d.struct({
     clamp: d.f32,
+    // Unfortunately, in structs, you cannot simply write d.arrayOf(d.f32)
```

Comment:
This is the first time we're seeing anything related to runtime-sized schemas, so I think this comment might be confusing here. Maybe we should introduce `d.arrayOf(d.f32)` as a neat shorthand later on?